config/crd/packet.crossplane.io_providers.yaml

```diff
@@ -3,10 +3,15 @@
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.3.0
   creationTimestamp: null
   name: providers.packet.crossplane.io
 spec:
   additionalPrinterColumns:
+  - JSONPath: .spec.projectID
```

Comment:
ProjectID will be removed if it doesn't make sense to live in the Provider resource.  It is not being used currently.

Reply:
ProjectID can now come from the resource, then fall back to the Provider, then fall back to the credentials.